rclcpp/include/rclcpp/experimental/subscription_intra_process.hpp

```diff
@@ -88,6 +89,15 @@ class SubscriptionIntraProcess : public SubscriptionIntraProcessBase
     if (RCL_RET_OK != ret) {
       throw std::runtime_error("SubscriptionIntraProcess init error initializing guard condition");
     }
+
+    TRACEPOINT(
```

Comment:
Please "include what you use", the tracetools header is not included in this file explicitly.

Reply:
Oops. Totally missed that. Fixed in c8fe4f1f09db798ba067857926d8ecd0efb48b48